src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,225 @@
+  <figcaption>
+    Example of Twitter with eight notifications and another app showing a flag
+    type badge.
+  </figcaption>
+</figure>
+
+The Badging API allows installed web apps to set an application-wide badge,
+shown in an operating-system-specific place associated with the application
+(such as the shelf or home screen).
+
+Badging makes it easy to subtly notify the user that there is some new activity
+that might require their attention, or to indicate a small amount of
+information, such as an unread count.
+
+Badges tend to be more user-friendly than notifications, and can be updated
+with a much higher frequency, since they don't interrupt the user. And,
+because they don't interrupt the user, they don't need the user's permission.
+
+### Possible use cases {: #use-cases }
+
+Examples of sites that may use this API include:
+
+* Chat, email, and social apps, to signal that new messages have arrived, or to
+  show the number of unread items.
+* Productivity apps, to signal that a long-running background task (such as
+  rendering an image or video) has completed.
+* Games, to signal that a player action is required (e.g., in Chess, when it
+  is the player's turn).
+
+## Current status {: #status }
+
+| Step                                       | Status                       |
+| ------------------------------------------ | ---------------------------- |
+| 1. Create explainer                        | [Complete][explainer]        |
+| 2. Create initial draft of specification   | [Complete][spec]             |
+| 3. Gather feedback & iterate on design     | [In progress](#feedback)     |
+| **4. Origin trial**                        | [**In progress**](#ot)       |
+| 5. Launch                                  | Not started                  |
+
+{% Aside 'codelab' %}
+See it in action
+{% endAside %}
+
+1. Using Chrome 73 or later on Windows or Mac, open the [Badging API demo][demo].
+2. When prompted, click **Install** to install the app, or use the Chrome
+   menu to install it.
+3. Open it as an installed PWA. Note, it must be running as an installed PWA (in
+   your task bar or dock).
+4. Click the **Set** or **Clear** button to set or clear the badge from the app
+   icon. You can also provide a number for the *Badge value*.
+
+{% Aside %}
+  While the Badging API *in Chrome* requires an installed app
+  with an icon that can actually be badged, you shouldn't
+  make calls to the Badging API dependent on the install state.
+  The Badging API can apply to *anywhere* a browser might want to show a badge,
+  so developers shouldn't make any assumptions about situations where
+  the browser will display them. Just call the API when it exists.
+  If it works, it works. If not, it simply doesn't.
+{% endAside %}
+
+
+## How to use the Badging API {: #use }
+
+Starting in Chrome 73, the Badging API is available as an origin trial
+for Windows (7+) and macOS.
+
+[Origin trials][ot-what-is] allow you to try out new features and give
```

Comment:
I believe Joe made an include for this.

```
{% include 'content/origin-trials.njk' %}
```

Reply:
That include doesn't include all of the stuff in this block. Plus, it doesn't allow us to link directly to the specific origin trial for this API.

Reply:
I just tried to include it, and it looks like some of the links in it are broken (uses markdown reference links). I'd also like to not have it as an aside. Will use for future docs if that's OK

Reply:
Since it's the same text as Joe's include it seems like we should deduplicate if possible. Checking the site... I don't think Joe's include is actually used anywhere yet so we could remove the Aside.